src/App.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+
+const chatDataList = chatMessages
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatDataList);
```

Comment:
Instead of setting state with chatDataList, you can set the state with chatMessages directly